R/bcdc_options.R

```diff
@@ -101,8 +100,31 @@ bcdc_single_download_limit <- function() {
     ))
 
     res <- cc$parse("UTF-8")
-    doc <- xml2::read_xml(res)
+    xml2::read_xml(res)
+
+  } else {
+    message("No access to internet")
+    invisible(FALSE)
+  }
+}
+
+bcdc_get_capabilities <- function() {
```

Comment:
I think a different name is in order here, since it's not getting all capabilities it's just getting the record attributes. `bcdc_get_records_from_capabilities()` or something like that?

Reply:
I had that same thought. What about `bcdc_get_wfs_records`?

Reply:
That works for me